modules/v2/variables.tf

```diff
@@ -55,8 +55,8 @@ variable "containers" {
       mount_path = string
     })), [])
     ports = optional(object({
-      name           = optional(string, "http1")
-      container_port = optional(number, 8080)
+      name           = optional(string)
```

Comment:
What name for the port is used if users doesn't define `name` but only configures `container_port`